src/System.Management.Automation/engine/ErrorPackage.cs

```diff
@@ -1790,6 +1792,8 @@ public interface IContainsErrorRecord
         ErrorRecord ErrorRecord { get; }
```

Comment:
```suggestion
        ErrorRecord? ErrorRecord { get; }
```

I found a lot of checks `ErrorRecord != null`. Some classes obviously return null. Some classes always return a value so we will annotate them appropriately.

Reply:
No this is correct. 
If you implement IContainsErrorRecord, you can deliver one. 
Lots of code depends on that.

Reply:
First link I opened was "_errorRecord = (ErrorRecord)info.GetValue("errorRecord", typeof(ErrorRecord));" but now I reviewed other implementations and agree with the design.

#closed.